config/settings.yml

```diff
@@ -5,6 +5,8 @@ chef:
 doorkeeper:
   administrators:
     - rainbowdash
+email_from_address: 'Chef Notifications <notifications@chef.io>'
```

Comment:
I'm not sure where settings.yml gets used in Rails, but I'm guessing based on the contents of this file, this is for testing.

At the top of this file it sets an endpoint that points to 33.33.33.10 - we should change this to be a true private ip. 192.168.*.*

I suspect this is a carry over from dev-vm and it's wrong in dev-vm too. If need be, lets figure out how to fix this in both (or even all) places. We can add a tasks to the backlog.

FYI, dev-vm has it wrong b/c for a long time vagrant had it wrong, but vagrant fixed it a while back.